sketch.js

```diff
@@ -33,57 +37,75 @@ function setup() {
   trainMap = mappa.tileMap(options);
   trainMap.overlay(canvas);
 
-  let maxSubs = 0;
-  let minSubs = Infinity;
+  createP('Select category!');
+  select = createSelect();
+  select.option('Subscribers');
+  select.option('Watch Time');
+  select.option('Views');
+  select.changed(processData);
+  createP(''); // look, I don't know how to use p5 properly, I used this to push the canvas a little down ;w;
```

Comment:
The best way to handle this is probably to make divs in `index.html` add a `style.css` and use `parent()` to place any generated DOM elements in the divs. The DOM elements could also be written into the HTML itself and accessed here with `select()`.